app/app/settings.py

```diff
@@ -796,7 +796,7 @@ def callback(request):
 
 WIKI_ACCOUNT_HANDLING = True
 WIKI_ACCOUNT_SIGNUP_ALLOWED = True
-WIKI_CACHE_TIMEOUT = True
+WIKI_CACHE_TIMEOUT = 600
```

Comment:
600 s? ms?

Reply:
> 600 s? ms?

Seconds to time out!

Reply:
600s seems like a lot. should it be more like 1s?

r u sure this will actually fix the issue?   IIRC from https://github.com/django-wiki/django-wiki/issues/1031 the issue was infinite recursion

Reply:
600s is standard setting + the live **gitcoin.co** indefenetely expecting for the **jquery 3.3.1** timing out the wiki!

![wiki1](https://user-images.githubusercontent.com/41552663/85330354-42677700-b4a2-11ea-969d-5e424880e3a2.gif)

Reply:
refer to [timeout settings](https://django-wiki.readthedocs.io/en/latest/settings.html#wiki.conf.settings.CACHE_TIMEOUT)

Reply:
yeah 600s is way too long, and i dont think its going to fix the issue anyway? I think the issue is within django-wiki

Reply:
> yeah 600s is way too long, and i dont think its going to fix the issue anyway? I think the issue is within django-wiki

Totally! Set 1s to timeout. We will find that fix for the wiki sooner or later